### ISSUE TITLE
Getting SIGILL with RUSTFLAGS='-C target-cpu=native'

### ISSUE THREAD

**Issue by learnbyexample** [sstadick/hck#25]
Here's the error message I got:

```
error: failed to run custom build command for `libc v0.2.98`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installMEiNJj/release/build/libc-d4f829098caa5901/build-script-build` (signal: 4, SIGILL: illegal instruction)
```

Works fine if `RUSTFLAGS` is empty. I'm on Ubuntu 20.04 LTS, x86_64 and `rustc` version is 1.47.0. I got similar issue with `frawk` before which was due to lack of `avx` support on my hardware - perhaps it is the same issue here as well?

---

Also, installing using `deb` file worked too. But, the name is `default.hck-linux-amd64.deb` in the release page and `hck-linux-amd64.deb` in the README.

**Comment by sstadick** [comment]
That's super odd! I haven't run into the native flag issue before. I also build on Ubuntu 20.04 when doing dev on an AMD cpu. 

None of the other install methods use the native flag at least, as you noted with the deb build. 

The deb file name should be fixed in the v0.5.3 release as well 👍

**Comment by sstadick** [comment]
I'm going to close this for now on the assumption that this won't be a common thing, but if it keeps popping up for users I'll adjust the docs.
